http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/RuntimeConfigTable.scala

```diff
@@ -53,11 +57,13 @@ class RuntimeConfigTable(tag: Tag) extends Table[RuntimeConfigRecord](tag, "RUNT
         val r = cloudService match {
           case CloudService.GCE =>
             diskSize match {
-              case Some(size) => RuntimeConfig.GceConfig(machineType, size)
+              case Some(size) => RuntimeConfig.GceConfig(machineType, size, bootDiskSize)
               case None =>
+                val bds =
+                  bootDiskSize.getOrElse(throw new SQLDataException("gce runtime with PD has to have a boot disk"))
                 persistentDiskId.fold(
-                  RuntimeConfig.GceWithPdConfig(machineType, None)
-                )(diskId => RuntimeConfig.GceWithPdConfig(machineType, Some(diskId)))
+                  RuntimeConfig.GceWithPdConfig(machineType, None, bds)
+                )(diskId => RuntimeConfig.GceWithPdConfig(machineType, Some(diskId), bds))
```

Comment:
having `bootDiskSize` in pubsub message is a bit annoying since this value won't actually being used in back Leo....created https://broadworkbench.atlassian.net/browse/IA-2014?atlOrigin=eyJpIjoiMTQ1YjBiMzJkNThiNDM1OWE5ODM0ZTE3YTQ0MjNkZTgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ to improve this in a separate PR